docs/topics/api/reviewers.rst

```diff
@@ -17,26 +17,30 @@ Subscribe
 ---------
 
 This endpoint allows you to subscribe the current user to the notification
-sent when a new listed version is submitted on a particular add-on.
+sent when a new version is submitted on a particular add-on.
 
     .. note::
         Requires authentication and the current user to have any
         reviewer-related permission.
 
-.. http:post:: /api/v4/reviewers/addon/(int:addon_id)/subscribe/
```

Comment:
Shouldn't we keep this endpoint documented? We could say that it's deprecated?

Reply:
Oh, right, I forgot to update the docs after restoring the old endpoint.

Reply:
0c3cb8a3b9

Reply:
Typo fix in f4efb60d04.